addon/CHANGELOG.md

```diff
@@ -0,0 +1,265 @@
+
```

Comment:
@chriskrycho so this one little bit tricky:

without the `*.md` files in the `addon` folder, npmjs.com package page shows missing readme.
You can see the problem in https://www.npmjs.com/package/ember-focus-trap and I've made [PR fixing the issue](https://github.com/josemarluedke/ember-focus-trap/pull/60).

Moving the `*.md` files to the `addon` folder makes `npm pack` and `npm publish` pick up those files and include into .tgz archive.

This PR also adds symlinks from the root folder to the `addon/` location so that GitHub still renders the repository overview correctly. It's important, that the files are physically present in the `addon/` folder instead of just symlinked into the package, as npm pack doesn't follow symlinks.

CONTRIBUTING.md is intentionally not moved, as it's only relevant in the context of the repository.

Reply:
Ahhhh, very good. Thanks for the clarification!